### DISCUSSION TITLE
Unable to pass Github Student verification.

### DISCUSSION THREAD

**Discussion by mathew4STAR** [community/community#65571] [GitHub Education]
### Hi everyone,

Hello,

I have tried multiple times, but I have been unable to pass Github student verification. My school has a school issued mail id, but it does not allow mails from other domains(school mail id's can only receive and send mails to other school mail id's). So I never get the verification email that github sends and hence can't use that.

My school id card also does not have a date.

I have also tried other things such as enrollment page, school issued schedules and timetables, none of them have worked.

Is their any other way to get verified?, does anyone have any ideas to get verified without these?

Thank you.

**Comment by arafajs** [comment]
- Contact your school's IT department to see if they can help you get your school email address to accept emails from other domains.
- Try using a different email address, such as a personal Gmail address, to receive the verification email from GitHub.
- If you have a dated school ID, you can try using that as proof of your student status. You can also try using a transcript, enrollment page, or school-issued schedule.
- If you have tried all of these things and you are still unable to verify your student status, you can contact GitHub support for help.
